slices/openjdk-25-jdk-headless.yaml

```diff
@@ -0,0 +1,164 @@
+package: openjdk-25-jdk-headless
+
+essential:
+  - openjdk-25-jdk-headless_copyright
+
+slices:
+  # OpenJDK binaries slice
+  core:
+    essential:
+      - libc6_libs
+      - openjdk-25-jre-headless_core
+      - zlib1g_libs
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/bin/javac:
+      /usr/lib/jvm/java-25-openjdk-*/bin/javadoc:
+      /usr/lib/jvm/java-25-openjdk-*/bin/javap:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jdeprscan:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jdeps:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jimage:
+      /usr/lib/jvm/java-25-openjdk-*/bin/serialver:
+      # used by javac
+      /usr/lib/jvm/java-25-openjdk-*/lib/ct.sym:
+
+  standard:
+    essential:
+      - openjdk-25-jdk-headless_core
+      - openjdk-25-jre-headless_standard
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jar:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jarsigner:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jcmd:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jdb:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jfr:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jhsdb:
+        arch:
+          - amd64
+          - arm64
+          - ppc64el
+          - riscv64
+      /usr/lib/jvm/java-25-openjdk-*/bin/jinfo:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jmap:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jnativescan:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jps:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jrunscript:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jshell:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jstack:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jstat:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jstatd:
+      /usr/lib/jvm/java-25-openjdk-*/bin/jwebserver:
+
+  # OpenJDK modules required to build the runtime image
+  modules:
```

Comment:
does it make sense to split the modules into self-contained slices?

PS is you want to include all these modules in your rock, then it's fine to preserve this catch-all `modules` slice.